Podfile

```diff
@@ -28,3 +28,11 @@ end
 target 'SnapshotTests' do
   pod 'AccessibilitySnapshot', '0.5.0'
 end
+
+post_install do |installer|
```

Comment:
Is this part of the changes for this PR as well?

Reply:
no more :)